### PR TITLE
fix package relative path

### PR DESCRIPTION
**Pull request by atishayv**
Fixes 
The adaptive expression library was not passing through babel .

## Description
Fix the relative path of all libraries for it to be compiled with babel.

## Testing
Tested with adaptive card library by replacing the minfied bundle

### CODE REVIEW

**Comment by atishayv** [comment]
@stevengum , @joshgummersall Can you please review this ?

File: libraries/adaptive-expressions-ie11/webpack.config.js [hunk 1]
```diff
@@ -33,14 +33,14 @@ module.exports = () => {
                         },
                     ],
                     include: [
-                        path.resolve(__dirname, './node_modules/adaptive-expressions'),
+                        path.resolve(__dirname, '../adaptive-expressions'),
```
Comment (joshgummersall) [inline libraries/adaptive-expressions-ie11/webpack.config.js:36]: Does require.resolve work? That should abstract this all away. You would just pass the name of the package.

**Review by joshgummersall — CHANGES_REQUESTED** [review]

File: libraries/adaptive-expressions-ie11/webpack.config.js (hunk 1 above)
Comment (atishayv) [reply libraries/adaptive-expressions-ie11/webpack.config.js:36]: Nope require.resolve doesnt work
`require.resolve('adaptive-expressions')`

**Comment by joshgummersall** [comment]
See https://github.com/microsoft/botbuilder-js/pull/3090 for the `require.resolve` technique.

**Pull request closed without merging**